app/templates/components/forms/events/view/create-discount-code.hbs

```diff
@@ -0,0 +1,94 @@
+  {{/if}}
+  <div class="field">
+    <label class="required">{{t 'Number of Discount Tickets'}}</label>
+    {{input name='number_of_discount_tickets' type='number' value=data.ticketsNumber min='0'}}
+  </div>
+  <div class="grouped inline fields">
+    <label class="required">{{t 'Status'}}</label>
+    <div class="field">
+      {{ui-radio name='status' label=(t 'Active') current='active' value='active'}}
+    </div>
+    <div class="field">
+      {{ui-radio name='status' label=(t 'Inactive') value='inactive'}}
+    </div>
+  </div>
+  <div class="inline field">
+    <label class="required">{{t 'Select Ticket(s) applied to the discount code'}}</label>
+    <div class="ui hidden divider"></div>
+    {{ui-checkbox name= 'all_ticket_types' label=(t 'Select all Ticket types') checked=allTicketsChecked onChange=(action (mut allTicketsChecked))}}
+  </div>
+  <div>
+    <div class="inline field">
+      {{ui-checkbox label=(t 'Show more options') class='toggle' checked=showMoreOptions onChange=(action (mut showMoreOptions))}}
+    </div>
+    {{#if showMoreOptions}}
+      <div>{{t 'Set the min and max quantity allowed to purchase per order (Optional)'}}</div>
+      <div class="{{unless device.isMobile 'fields'}}">
+        <div class="wide field {{if device.isMobile 'sixteen' 'five'}}">
+          <label>{{t 'Min'}}</label>
+          {{input type='number' name='min_order' min='0' value=data.minQuantity}}
+        </div>
+        <div class="wide field {{if device.isMobile 'sixteen' 'five'}}">
+          <label>{{t 'Max'}}</label>
+          {{input type='number' name='max_order' min='0' value=data.maxQuantity}}
+        </div>
+      </div>
+      <div class="fields">
+        <div class="wide field {{if device.isMobile 'sixteen' 'five'}}">
+          <label>{{t 'Valid from'}}</label>
+          {{widgets/forms/date-picker id='start_date' value=data.validFromDate}}
+          <div class="ui hidden divider"></div>
+          {{widgets/forms/time-picker id='start_time' value=data.validFromTime}}
+        </div>
+        <div class="wide field {{if device.isMobile 'sixteen' 'five'}}">
+          <label>{{t 'Expires on'}}</label>
+          {{widgets/forms/date-picker id='end_date' value=data.validTillDate}}
+          <div class="ui hidden divider"></div>
+          {{widgets/forms/time-picker id='end_time' value=data.validTillTime}}
+        </div>
+      </div>
+    {{/if}}
+    <div class="field">
+      <label>{{t 'Discount Link'}}</label>
+      <div class="ui action input fluid">
+        <input type="text" value="{{discountLink}}" readonly>
+        {{#copy-button
+          clipboardText=discountLink
+          class='ui blue labeled icon copy button'}}
+          <i class="copy icon"></i>
+          {{t 'Copy'}}
+        {{/copy-button}}
+      </div>
+    </div>
+  </div>
+  <div class="ui hidden divider"></div>
+  {{#link-to 'events.view.tickets.discount-codes'}}
+    <button class="ui button">
+      {{t 'Cancel'}}
+    </button>
```

Comment:
@utkarshnath remove this `cancel` button not needed here since it is page, not a model